src/core/DelayedFunctionScheduler.js

```diff
@@ -126,7 +129,10 @@ getJasmineRequireObj().DelayedFunctionScheduler = function() {
 
         currentTime = newCurrentTime;
 
-        var funcsToRun = scheduledFunctions[currentTime];
+        var funcsToRun = scheduledFunctions[currentTime].sort(function (a, b) {
```

Comment:
JavaScript's `sort` is not guaranteed to be stable, so this potentially causes functions that are scheduled for the same time to run in the wrong order. I think that's why the tests [fail in PhantomJS](https://travis-ci.org/jasmine/jasmine/jobs/286512581) even though they pass in other browsers.

I see that that the same test fails in other browsers if this `sort` is removed. If that's the only reason for the `sort`, a better approach might be to change the test to explicitly schedule the one function slightly after the other.

Reply:
@sgravrock Oh I think you're right :) I'll fix this up today. Sorry I fired off this PR late at night and sort of forgot about it.